sips/sip-config-waiting-period.md

```diff
@@ -0,0 +1,84 @@
+---
+sip: 116
+title: Conditionally skipping fee reclamation
+status: Proposed
+author:
+discussions-to:
+created: 2020-02-25
+---
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Simply describe the outcome the proposed changes intends to achieve. This should be non-technical and accessible to a casual community member.-->
+
+Allow waiting period to be set to zero and avoid unecessary fee reclamation functionality.
```

Comment:
`s/unecessary/unnecessary`

Reply:
I would say "bypass fee reclamation requirement when the waiting period is set to 0"